research/object_detection/meta_architectures/context_rcnn_lib_v2.py

```diff
@@ -0,0 +1,217 @@
+# Lint as: python3
```

Comment:
Let's name this file context_rcnn_lib_tf2.py

Reply:
I see, I thought usually the test files for tf 2 have _tf2 names but the tf2 versions of the libs have _v2 (model_lib_v2.py)?